src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,13 @@
+<Configuration>
+   <Appenders>
+      <Console name="STDOUT" target="SYSTEM_OUT">
+         <PatternLayout pattern="%5p | %d | %L | %-40c{1} | %m%n"/>
+      </Console>
+   </Appenders>
+   <Loggers>
+      <Logger name="org.apache.log4j.xml" level="info"/>
+      <Root level="info">
+         <AppenderRef ref="STDOUT"/>
+      </Root>
+   </Loggers>
+</Configuration>
```

Comment:
Try this to restore the file appender as well:
```xml
<Configuration>
   <Properties>
      <Property name="filePattern">488Logs/default</Property>
   </Properties>
   <Appenders>
      <Console name="STDOUT" target="SYSTEM_OUT">
         <PatternLayout pattern="%5p | %d | %L | %-40c{1} | %m%n"/>
      </Console>
      <RollingFile name="File" fileName="${filePattern}.log" filePattern="${filePattern}-%i.log">
         <PatternLayout pattern="%5p | %d | %L | %-40c{1} | %m%n"/>
         <Policies>
            <OnStartupTriggeringPolicy/>
         </Policies>
         <DefaultRolloverStrategy max="10"/>
      </RollingFile>
   </Appenders>
   <Loggers>
      <Logger name="org.apache.log4j.xml" level="info"/>
      <Root level="debug">
         <AppenderRef ref="STDOUT"/>
         <AppenderRef ref="File"/>
      </Root>
   </Loggers>
</Configuration>
```

Reply:
This is me remembering from early 2023, but I think this was intentional since AdvantageKit is already handling all the "writing to disk" that the FileAppender would have done.